### DISCUSSION TITLE
Can't download the Examples folder?

### DISCUSSION THREAD

**Discussion by quinnsea** [nathanhoad/godot_dialogue_manager#269] [Q&A] [answered]
Hey, super new to Godot and downloading addons on Git! Whenever I download this, the examples folder doesn't show up in the ZIP that I get. I tried downloading both through the most recent (2.2.0) release announcement and the "Download ZIP" button from the main branch. For my project, I'd really like to check out the visual novel example specifically but it's just not coming through. I have the example_balloon in the dialogue_manager folder, but that's it.

**Comment by nathanhoad** [comment, marked as answer]
Only the `addons` folder is included when download a zip of the repository. To get the examples folder you'll need to clone the repository.

**Reply by quinnsea** [reply]
Gotcha, thank you so much!
